_rules/SC1-3-1-aria-required-owned-element.md

```diff
@@ -0,0 +1,243 @@
+---
+name: ARIA required owned elements
+
+description: |
+	The rule checks that a role has at least one of its required owned elements.
+
+success_criterion:
+- 1.3.1 # Info and Relationships (A)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Audrey Maniez
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML or SVG element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has an explicit [semantic role](#semantic-role) that has [required owned elements](https://www.w3.org/TR/wai-aria-1.1/#mustContain) listed for that role in [WAI-ARIA](https://www.w3.org/TR/wai-aria).
+
+### Expectation
+
+For each test target at least one instance of one [required owned element](https://www.w3.org/TR/wai-aria-1.1/#mustContain) listed for that role in [WAI-ARIA](https://www.w3.org/TR/wai-aria) is present as a [semantic role](#semantic-role) for an element [owned](https://www.w3.org/TR/wai-aria-1.1/#dfn-owned-element) by the test target.
+
+**Note:**
+- When a widget is missing required owned elements due to script execution or loading, authors [MUST]((w3.org/TR/wai-aria-1.1/#mustContain)) mark a containing element with `aria-busy` equal to `true`.
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility Support
+
+When a required owned element is not an immediate child, but rather a nested descendant, certain AT have issues recognizing the owned element.
```

Comment:
Is it correct to say:
[[
This rule relies on assistive technologies that can recognize the owned element when it is a nested descendant, rather than an immediate child.
]]

Just an attempt at making it more specific and less open to interpretation of what "issues" are.

Reply:
This doesn't have anything to do with support. This is how its supposed to work. The definition of "owned element" is pretty misleading. An element isn't considered to be part of a group unless its 1) a child, 2) referenced by aria-owns or 3) a descendent, with every element between it and the owner marked as role=presentation/none.

Reply:
Per this https://github.com/auto-wcag/auto-wcag/pull/245#discussion_r222282853, I simply don't think that that is true; it has everything to do with support as it _is_ how it's supposed to work, both according to ARIA itself and the associated authoring practices.

Reply:
Something to discuss during the rule writing session. I will add to agenda.

Reply:
Or maybe on Thursday so that we can complete it earlier?

Reply:
Newest Accessibility Support content from sibling rule https://github.com/auto-wcag/auto-wcag/pull/255: 
"This rule relies on assistive technologies to recognize owned elements, as defined by [WAI-ARIA 1.1](https://www.w3.org/TR/wai-aria). This includes when they are nested descendants that are not immediate children. However, some assistive technologies do not recognize owned elements that are not immediate children, unless workarounds are used."

This discussion point is equally relevant for both these rules.

Reply:
Updated to above comment, and have added to agenda for wider discussion.